Blockzilla/HomeView.swift

```diff
@@ -19,15 +19,25 @@ class HomeView: UIView {
     private let tipTitleLabel = SmartLabel()
     private let tipDescriptionLabel = SmartLabel()
     private let shieldLogo = UIImageView()
+    private let textLogo = UIImageView()
     
     let toolbar = HomeViewToolbar()
     let trackerStatsShareButton = UIButton()
     
+    override func layoutSubviews() {
```

Comment:
Rather than overriding the layoutSubviews function and increasing overhead every time it's called, it's probably better to just override the function called when the device rotates. [This link may be helpful](https://stackoverflow.com/a/41724901).

Reply:
ok